src/basic/special.h

```diff
@@ -87,6 +87,8 @@
 #define SPECIAL_REMOUNT_FS_SERVICE "systemd-remount-fs.service"
 #define SPECIAL_VOLATILE_ROOT_SERVICE "systemd-volatile-root.service"
 #define SPECIAL_UDEVD_SERVICE "systemd-udevd.service"
+#define SPECIAL_GROWFS_SERVICE "systemd-growfs@.service"
```

Comment:
This is nice. I think we should rework the other places that generate a unit in full to have a common stub too. It's confusing to users if a unit doesn't have a static file anywhere.

Reply:
which one do you have in mind?

Reply:
Basically anything that uses `generator_open_unit_file` would be a candidate. But I didn't look into it too closely. If the unit is not templated, we probably can't do that.